packages/cli/src/services/orchestration/handleCommandMessage.ts

```diff
@@ -0,0 +1,32 @@
+import { LoggerProxy } from 'n8n-workflow';
+import { messageToRedisServiceCommandObject } from './helpers';
+import Container from 'typedi';
+import { License } from '@/License';
+
+// this function handles commands sent to the MAIN instance. the workers handle their own commands
+export async function handleCommandMessage(messageString: string, uniqueInstanceId: string) {
+	const message = messageToRedisServiceCommandObject(messageString);
+	if (message) {
+		if (
+			message.senderId === uniqueInstanceId ||
+			(message.targets && !message.targets.includes(uniqueInstanceId))
+		) {
+			LoggerProxy.debug(
+				`Skipping command message ${message.command} because it's not for this instance.`,
+			);
+			return message;
+		}
+		switch (message.command) {
+			case 'reloadLicense':
+				const instanceId = Container.get(License).instanceId;
+				if (instanceId) {
+					await Container.get(License).init(instanceId, 'main');
```

Comment:
can we use the License SDK's `reload()` function instead?

Reply:
If that effectively does the same, sure!